Automattic/wp-calypso
# Move the woocommerce installer transfer step to its own wrapper

#### Changes proposed in this Pull Request

* Make it possible to nest steps under other step paths
* Move the woocommerce installer transfer step to its own step wrapper
* Bug fix on the progress bar to make sure it stalls at .95 instead of fluctuating between .95 and 1

#### Testing instructions

* Confirm the transfer flow still works http://calypso.localhost:3000/start/woocommerce-install/confirm?siteSlug=wooptestupgrade.wordpres\s.com
* Confirm the normal start flow still works http://calypso.localhost:3000/start



Related to #
